src/View/Artist/search.html.twig

```diff
@@ -0,0 +1,28 @@
+{% extends 'layout.html.twig' %}
+
+{% block content %}
+	<div>
+		<p>Trouve des artistes</p>
+	</div>
+	{% include 'Artist/_searchArtiste.html.twig' %}
+
+	<div class="container marketing">
+
+		<div class="row">
+			{% for artist in artists %}
+				<div class="col-lg-4 text-align-center">
+					<img src="{{ artist.image }}" class="bd-placeholder-img rounded-circle" width="140" height="140" xmlns="http://www.w3.org/2000/svg" role="img" aria-label="Placeholder: 140x140" preserveaspectratio="xMidYMid slice" focusable="false">
```

Comment:
The attribute html width and height should be in the css file

Reply:
update on style.css